src/savings-pool-client.ts

```diff
@@ -62,8 +58,77 @@ export default class SavingsPoolClient {
    * @returns The operation hash
    */
   public async redeem(lpTokenAmount: BigNumber): Promise<TransactionOperation | TransactionWalletOperation> {
-    const liquidityPoolContract = await this.tezos.wallet.at(this.savingsPoolAddress)
+    const savingsPoolContract = await this.tezos.wallet.at(this.savingsPoolAddress)
     const sendArgs = { amount: 0, mutez: true }
-    return await liquidityPoolContract.methods['redeem'](lpTokenAmount).send(sendArgs)
+    return await savingsPoolContract.methods['redeem'](lpTokenAmount).send(sendArgs)
+  }
+
+  /**
+   * Get the interest rate of the pool.
+   */
+  public async getInterestRateAPY(): Promise<BigNumber> {
+    const savingsPoolContract = await this.tezos.wallet.at(this.savingsPoolAddress)
+    const savingsPoolStorage: any = await savingsPoolContract.storage()
+    const interestRate = savingsPoolStorage.interestRate
+    return interestRateToApy(interestRate)
+  }
+
+  /** 
+   * Get the size of the pool in kUSD, accounting for the current time.
+   */
+  public async getPoolSize(): Promise<BigNumber> {
+    const savingsPoolContract = await this.tezos.wallet.at(this.savingsPoolAddress)
+    const savingsPoolStorage: any = await savingsPoolContract.storage()
+    const poolSize = savingsPoolStorage.underlyingBalance
+    const lastInterestUpdate = savingsPoolStorage.lastInterestCompoundTime
+    const interestRate = savingsPoolStorage.interestRate
+
+    // TODO(keefertaylor): Similiar to the API in StableCoin client. Consider deduping / refactoring.
+    const time = new Date()
```

Comment:
Same request with any dates, can you make these optional params so I can hook up things like internal tick/clocks used in the frontend?